src/ExactDiagonalization/algorithms.jl

```diff
@@ -15,7 +15,7 @@ non-hermitian matrices.
     will be instantiated. This is typically faster and recommended for small matrices,
     but requires more memory. If `true`, the matrix is not instantiated, which is useful for
     large matrices that would not fit into memory. The calculation will parallelise using
-    threading and MPI if available by making use of [`PDVec`](@ref).
+    threading if available by making use of [`OperatorAsMap`](@ref).
```

Comment:
```suggestion
    threading if available by making use of [`LinearOperator`](@ref).
```
Maybe not yet very clear for the reader. 

I'm wondering whether we should keep around the option of working with `PDVec`s? I agree that the new way should be the default because it is faster and uses less memory.